Translation/ConfigFactory.php

```diff
@@ -43,11 +57,20 @@ public function getBuilder($name)
         return $this->builders[$name];
     }
 
+    /**
+     * @param $name
+     * @param $locale
+     * @return mixed
+     */
     public function getConfig($name, $locale)
     {
         return $this->getBuilder($name)->setLocale($locale)->getConfig();
```

Comment:
Hm... will this work? `PhpParser\Builder` does not have a setLocale...  Am I missing something?